pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -905,6 +905,8 @@ func (qjm *XController) getAggregatedAvailableResourcesPriority(unallocatedClust
 			continue
 		} else if !value.Status.CanRun {
 			klog.V(11).Infof("[getAggAvaiResPri] %s: Skipping adjustments for %s since it can not run.", time.Now().String(), value.Name)
```

Comment:
Since we are changing the logic in this case, we need to update the log message. The log message should be that we adjust the available resources by adding the resources consumed by the AW.

Reply:
Thanks, Done